pkg/manager/member/tidb_member_manager.go

```diff
@@ -898,6 +999,28 @@ func (m *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	return nil
 }
 
+func (m *tidbMemberManager) GetDBUrl(tc *v1alpha1.TidbCluster) (string, error) {
+	return fmt.Sprintf("root:@tcp(%s-tidb.%s.svc:4000)/?charset=utf8mb4,utf8&multiStatements=true", tc.Name, tc.Namespace), nil
+}
+
+func (bo *tidbMemberManager) OpenDB(ctx context.Context, dsn string) (*sql.DB, error) {
```

Comment:
Can we reuse the one in https://github.com/pingcap/tidb-operator/blob/master/cmd/backup-manager/app/util/util.go#L117?

Reply:
This can be removed.